### ISSUE TITLE
Add support for SonarQube scanning to Golang quickstarter

### ISSUE THREAD

**Issue by metmajer** [opendevstack/ods-project-quickstarters#381]
**Is your feature request related to a problem? Please describe.**
Currently, ODS does not deliver code quality metrics for Golang components since the corresponding SonarQube plugin is not available.

**Describe the solution you'd like**
Support the SonarQube scanner for Golang (via https://docs.sonarqube.org/latest/analysis/languages/go/).

**Describe alternatives you've considered**
~A clear and concise description of any alternative solutions or features you've considered.~

**Additional context**
~Add any other context or screenshots about the feature request here.~

**Comment by michaelsauter** [comment]
Good point, but I think we need to answer some more general questions here:

* Should we deliver all plugins with ODS? And how do we do that? As far as I know, there is no nice `plugins.txt` like one has with Jenkins that describes which plugins SonarQube should install. Putting the JAR files into the repo is not nice and might also be tricky from a license perspective.
* How do we handle private / optional quickstarters? How can they register the plugins? Or do we require an admin to install the plugins manually then? BTW: Right now, when you see languages other than Java supported in SQ I think it is because some admin at some point in time installed those plugins via the marketplace ...

**Comment by metmajer** [comment]
Got it. I understand that SQ lacks a convenient plugin infrastructure. If there's no alternative to installing a plugin via the Marketplace, or manually, as mentioned [here](https://docs.sonarqube.org/latest/setup/install-plugin/), then our options are indeed very limited.

**Comment by clemensutschig** [comment]
@michaelsauter - As we ship go as quickstarter.. we should add the plugin.. into 1.2

**Comment by michaelsauter** [comment]
@clemensutschig We do not ship the other plugins either (e.g. TypeScript, Python) ...

**Comment by michaelsauter** [comment]
OK, I learned we actually DO ship the plugins as JAR files now. So we should do the same for Go. I'm transferring this issue to ods-core now and will add it tomorrow.
